### DISCUSSION TITLE
Homens São Simples

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#490] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/homens-sao-simples/

**Comment by the-coffee-monk** [comment]
Foi só isso que eu vi:



[![null](<https://camo.githubusercontent.com/a19d963e6f852f66b030a49e6868184836b74383366b526c23b3ddfcdbd5f76f/68747470733a2f2f692e696d6775722e636f6d2f59655274374a342e676966>)](<https://camo.githubusercontent.com/a19d963e6f852f66b030a49e6868184836b74383366b526c23b3ddfcdbd5f76f/68747470733a2f2f692e696d6775722e636f6d2f59655274374a342e676966>)

**Comment by maryon-telvanni** [comment]
> [The Coffee Monk](</perfil/?user=the-coffee-monk>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/homens-sao-simples/#comment_1173902163>):

> Foi só isso que eu vi:[![null](<https://camo.githubusercontent.com/a19d963e6f852f66b030a49e6868184836b74383366b526c23b3ddfcdbd5f76f/68747470733a2f2f692e696d6775722e636f6d2f59655274374a342e676966>)](<https://camo.githubusercontent.com/a19d963e6f852f66b030a49e6868184836b74383366b526c23b3ddfcdbd5f76f/68747470733a2f2f692e696d6775722e636f6d2f59655274374a342e676966>)

Nuss! Mas que a cabeça do cara girou 360º com esse tapa

**Comment by brunoaugusto** [comment]
![null](<https://mario.wiki.gallery/images/4/4b/Peach_Frying_Pan.png>)

**Comment by one-eyed-willy** [comment]
![null](<https://i.imgur.com/dAgO7ii.png>)
